charts/images.yaml

```diff
@@ -5,7 +5,7 @@
 ---
 images:
 - name: alpine
-  repository: eu.gcr.io/gardener-project/3rd/alpine
+  repository: europe-docker.pkg.dev/gardener-project/releases/3rd/alpine
```

Comment:
Same as https://github.com/gardener/gardener-extension-registry-cache/pull/113#discussion_r1426264389

Reply:
addressed in: https://github.com/gardener/ci-infra/pull/1066